src/test/kotlin/apply/application/AssignmentServiceTest.kt

```diff
@@ -0,0 +1,101 @@
+package apply.application
+
+import apply.createAssignment
+import apply.createAssignmentRequest
+import apply.createEvaluationTarget
+import apply.createMission
+import apply.domain.assignment.AssignmentRepository
+import apply.domain.evaluationtarget.EvaluationStatus
+import apply.domain.evaluationtarget.EvaluationTargetRepository
+import apply.domain.mission.MissionRepository
+import apply.domain.mission.getById
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+import support.test.UnitTest
+import java.time.LocalDateTime
+
+@UnitTest
+class AssignmentServiceTest {
+    @MockK
+    private lateinit var assignmentRepository: AssignmentRepository
+
+    @MockK
+    private lateinit var missionRepository: MissionRepository
+
+    @MockK
+    private lateinit var evaluationTargetRepository: EvaluationTargetRepository
+
+    private lateinit var assignmentService: AssignmentService
+
+    @BeforeEach
+    internal fun setUp() {
+        assignmentService = AssignmentService(assignmentRepository, missionRepository, evaluationTargetRepository)
+    }
+
+    @Test
+    fun `과제 제출물을 생성한다`() {
+        every { assignmentRepository.existsByUserIdAndMissionId(any(), any()) } returns false
+        every { missionRepository.getById(any()) } returns createMission()
+        every {
+            evaluationTargetRepository.findByEvaluationIdAndUserId(any(), any())
+        } returns createEvaluationTarget()
+        every { assignmentRepository.save(any()) } returns createAssignment()
+
+        assertDoesNotThrow { assignmentService.create(1L, 1L, createAssignmentRequest()) }
+    }
+
+    @Test
+    fun `과제 제출 기간이 아니면 생성할 수 없다`() {
+        every { assignmentRepository.existsByUserIdAndMissionId(any(), any()) } returns false
+        every { missionRepository.getById(any()) } returns createMission(
+            startDateTime = LocalDateTime.now().minusDays(2), endDateTime = LocalDateTime.now().minusDays(1)
+        )
+        every {
+            evaluationTargetRepository.findByEvaluationIdAndUserId(any(), any())
+        } returns createEvaluationTarget()
+        every { assignmentRepository.save(any()) } returns createAssignment()
+
+        assertThrows<IllegalStateException> { assignmentService.create(1L, 1L, createAssignmentRequest()) }
+    }
+
+    @Test
+    fun `이미 제출한 이력이 있는 경우 새로 제출할 수 없다`() {
+        every { assignmentRepository.existsByUserIdAndMissionId(any(), any()) } returns true
+        every { missionRepository.getById(any()) } returns createMission()
+        every {
+            evaluationTargetRepository.findByEvaluationIdAndUserId(any(), any())
+        } returns createEvaluationTarget()
```

Comment:
스타일 문제라서 가볍게 봐주시면 좋을 것 같아요 👀   (다시보니 58~60번 줄도 같은 상황이네요!)
```suggestion
        every { evaluationTargetRepository.findByEvaluationIdAndUserId(any(), any()) } returns createEvaluationTarget()
```


<img src="https://user-images.githubusercontent.com/47850258/136709830-5a661e52-abff-42c0-8968-d10732d9e6d1.png" width="800" height="200">

Reply:
해당 파일 전반을 확인하고, 수정 마쳤습니다. 
꼼꼼한 확인 감사합니다!!! 👍👍👍